### PR TITLE
Fix #595 "View Console Log", Add scrolling, Launch Job Folder in file browser.

### LINKED ISSUE #595: [FEATURE] "View Output" on Jobs panel is ambigious
A lot of users (myself included initially) will think that View Output means to review the output boxes of the pipeline not console output, might want to consider changing it to something else "View Console Log" or related

Definitely a good feature to have though

### LINKED ISSUE #601: [FEATURE] Add scroll bar on view console output in jobs panel

### PR DESCRIPTION
**Pull request by subdavis**
Screenshot from 2021-03-08 12-43-35

fixes #595
fixes #601

### CODE REVIEW

**Review by BryonLewis — DISMISSED** [review]
This is just me being a pain.  Could we possibily:
`@click="visibleOutput = visibleOutput !== job.job.key ? job.job.key : null"`
or something similar to be able to toggle close an open console, maybe a text change in the process as well.
I just know that I've subconsciously clicked again expecting it to work more like a vuetify expansion panel.  Not a big deal if out of scope and obviously not necessary right now.

**Comment by subdavis** [comment]
You should know by now that I can never just do one thing.

File: client/platform/desktop/frontend/components/Jobs.vue [hunk 1]
```diff
@@ -101,6 +122,15 @@ export default defineComponent({
                           {{ job.job.datasetIds.join(', ') }}
                         </td>
                       </tr>
+                      <tr>
+                        <td>work dir</td>
+                        <td
+                          class="selectable"
+                          @click="openPath(job.job)"
+                        >
+                          show in file manager
```
Comment (BryonLewis) [inline client/platform/desktop/frontend/components/Jobs.vue:131]: Is this intentionally styled to be somewhat hidden?  It doesn't look like a clickable area like the `Console Log` it's regular text that has a cover state.

**Review by BryonLewis — APPROVED** [review]
I'm good with whatever you decide for the styling of the 'file manager' button, I just noticed that it doesn't standout but it may not need to.  Thanks for the button, it is much appreciated.

File: client/platform/desktop/frontend/components/Jobs.vue (hunk 1 above)
Comment (subdavis) [reply client/platform/desktop/frontend/components/Jobs.vue]: Better?

Screenshot from 2021-03-08 17-40-23

**Review by BryonLewis — APPROVED** [review]

**Pull request merged**